cli/template/extras/src/server/api/trpc/base.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * YOU PROBABLY DON'T NEED TO EDIT THIS FILE
```

Comment:
this comment may be a bit "harsh"? Like you should probably change it but not a lot?

Reply:
What cases would you want to change it? Might be worth calling them out specifically

Reply:
2 things pop into mind initially:

1. when changing context (e.g. you need the request headers for some procedures)
2. adding a new middleware/procedure such as `adminProcedure`

Reply:
Knowing that you can just leave this file alone at the start makes the tRPC boilerplate a bit less scary (it's quite scary), but I agree that there might be a better way to express this

Reply:
Any idea?

Reply:
This comment feels fair to me as a t3-stack beginner. Unlike a very explicit `Do not edit` [comment in Golang-land](https://github.com/golang/go/issues/41196), this one says that you _probably_ don't need to edit this file but you can.

Reply:
How about something like:

```ts
/**
 * YOU PROBABLY DON'T NEED TO EDIT THIS FILE, UNLESS:
 * 1. You want to modify request context (see Part 1)
 * 2. You want to create a new middleware or type of procedure (see Part 3)
 */
```

Could also links to tRPC docs but I'm leaning not necessary because they're already included further down